src/routes/index.tsx

```diff
@@ -132,7 +105,8 @@ const Routes = (): React.ReactElement => {
         render={() => {
           // Routes with a shortName prefix
           const validShortName = setChainIdFromUrl(pathname)
-          return validShortName ? <SafeContainer key={key} /> : <Redirect to={WELCOME_ROUTE} />
+          const safeAddress = extractSafeAddress()
+          return validShortName ? <SafeContainer key={safeAddress} /> : <Redirect to={WELCOME_ROUTE} />
```

Comment:
Would be worthwhile adding a comment regarding the `key` since the hook is now removed.